app/build.gradle

```diff
@@ -26,7 +26,7 @@ android {
         targetSdkVersion 32
         versionCode 11 // This versionCode is "frozen" for local builds. For "release" builds we
                        // override this with a generated versionCode at build time.
-        versionName "106.0"
+        versionName Config.generateDebugVersionName()
```

Comment:
If you can, a similar-ish comment for the `versionCode` would be good to add here even though it wasn't because of your changes - took me a few seconds to realize this is an overridden property as well (without having looked at Christian's comments above).

Reply:
Added a comment